cloud/pkg/router/listener/http.go

```diff
@@ -78,7 +78,11 @@ func (rh *RestHandler) RemoveListener(key interface{}) {
 func (rh *RestHandler) matchedPath(uri string) (string, bool) {
 	var candidateRes string
 	rh.handlers.Range(func(key, value interface{}) bool {
-		pathReg := key.(string)
+		pathReg, ok := key.(string)
+		if !ok {
+			klog.Errorf("key type %T error", key)
+			return true
```

Comment:
return false

Reply:
The return value should be `true`, according to the Range function.
![image](https://user-images.githubusercontent.com/49225405/175442157-cdc656f9-cc5c-4314-956d-47f7df9e6ced.png)